projects/packages/forms/src/dashboard/inbox/index.js

```diff
@@ -103,6 +108,12 @@ const Inbox = () => {
 		} );
 	}, [ currentPage, fetchResponses, query ] );
 
+	useEffect( () => {
```

Comment:
So, here's an idea, how about instead of dealing with state and everything we check on the backend if you have any feedback posts and return a separate config variable or simply redirect to the landing page if empty?

As is, there are some glitches. When you select a tab or a filter that happens to have no responses but you have plenty of responses in general, it'd still redirect you to the landing page. We can't deal with all of it on the frontend alone and doing it on the backend would also make it more responsive by immediately showing you what you need to see :)